.github/actions/cancel_azureml_jobs/cancel_aml_jobs.py

```diff
@@ -0,0 +1,42 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import os
+
+from azureml._restclient.constants import RunStatus
+from azureml.core import Experiment, Run, Workspace
+from azureml.core.authentication import ServicePrincipalAuthentication
+
+
+def cancel_running_and_queued_jobs() -> None:
+    print("Authenticating")
+    auth = ServicePrincipalAuthentication(
+        tenant_id='72f988bf-86f1-41af-91ab-2d7cd011db47',
+        service_principal_id=os.environ["HIML_SERVICE_PRINCIPAL_ID"],

```

Comment:
Maybe use ENV_SERVICE_PRINCIPAL_ID, ENV_SERVICE_PRINCIPAL_PASSWORD, ENV_TENANT_ID instead of hard coded strings?

Reply:
Oh can you actually import from hi-ml-azure at this level?

Reply:
If I want that script to run with minimum dependencies, no. I wanted to ensure that this job is as fast as somewhat possiblee.